src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -1117,7 +1119,9 @@ public String handle(
 						.computeIfAbsent(serialNumber, k -> new DeviceConfig())
 						.apply(apConfig);
 				});
-				configManager.wakeUp();
+				final String zone =
+					deviceDataManager.getDeviceZone(serialNumber);
+				configManager.wakeUp(zone);
```

Comment:
Hm.... perhaps we want to actually have a separate call for specific devices. Put a TODO here for now since it doesn't make sense to trigger zone update for a single AP change.